packages/spanner/src/transaction-request.js

```diff
@@ -203,11 +203,11 @@ TransactionRequest.prototype.createReadStream = function(table, query) {
 
   if (query.keys) {
     reqOpts.keySet = {
-      keys: [
-        {
-          values: arrify(query.keys).map(codec.encode)
-        }
-      ]
+      keys: arrify(query.keys).map(function(key) {
```

Comment:
@lukesneeringer That is the idea! Since we don't use monotonically increasing integers for data-distribution reasons, a composite key is _the_ way to do it.